src/Reveal.Sdk.Dom/Data/DataSources/SnowflakeDataSource.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Reveal.Sdk.Dom.Data
 {
-    public class SnowflakeDataSource : ProcessDataSource
```

Comment:
Why was the base class changed to remove the process properties?

Reply:
The `SchemaDataSource` is inherited from `ProcessDataSource`, so this will remains properties from Process data source.
The snowflake have `schemas` so I changed it to be a child of `SchemaDataSource`